data/xsum/facebook-bart-large-xsum-config.json

```diff
@@ -82,7 +82,7 @@
   "static_position_embeddings": false,
   "student_decoder_layers": null,
   "student_encoder_layers": null,
-  "sumtool_name": "facebook/bart-large-xsum",
+  "sumtool_id": "facebook-bart-large-xsum",
```

Comment:
so this `sumtool_id` is effectively the "foreign key" for the name of the metric, summary JSON files, correct? It _should_ correspond to the name of this config file, but it really could be anything? Is that correct?